Makefile.PL

```diff
@@ -278,34 +270,33 @@ my %optional_features = (
         }
     },
     inbound_email => {
-        prereqs => {
+        description => 'Inbound Email' prereqs => {
             runtime => {
                 requires => {
                     'Email::MIME::Attachment::Stripper' => 0,
-                    'Email::Reply'                => 0,
+                    'Email::Reply'                      => 0,
                 }
             }
         },
-        description => 'Inbound Email'
     },
     patch_viewer => {
         description => 'Patch Viewer',
-        prereqs => { runtime => { requires => { PatchReader => '0.9.6' } } }
+        prereqs     => { runtime => { requires => { PatchReader => '0.9.6' } } }
     },
     rest => {
         description => 'REST Interface',
         prereqs     => {
             runtime => {
                 requires => {
                     'Test::Taint' => '1.06',
-                    'JSON::RPC' => '==1.01',
+                    'JSON::RPC'   => '==1.01',
```

Comment:
`==` probably isn't required

Reply:
I don't want to deal with a newer JSON::RPC. This syntax makes it impossible to run with a newer JSON::RPC which is precisely what I want. :)